src/molecule/data/molecule.json

```diff
@@ -225,7 +225,7 @@
         "network_mode": {
           "anyOf": [
             {
-              "enum": ["bridge", "host", "none"],
+              "enum": ["bridge", "default", "host", "none"],
```

Comment:
If this is directly passed to docker, then this can be an arbitrary string (name of a docker network) and shouldn't be limited to a specific selection of networks that might or might not exist depending which networks the user created or deleted.

Reply:
I could be wrong but this is the mode and not the name of the network no?

Reply:
You are right so basically this should be used instead: 

```JSON
        "network_mode": {
          "title": "Network Mode",
          "type": "string"
        },
```

Reply:
I've raised https://github.com/ansible-community/molecule/pull/3725 as a fix.